docs/app/pages/components/components-sections/panels/accordion/accordion.component.html

```diff
@@ -0,0 +1,37 @@
+<accordion [closeOthers]="true">
+
+    <accordion-group *ngFor="let group of groups" [(isOpen)]="group.open">
+
+        <ng-container accordion-heading>
+            <i class="hpe-icon pull-right" [class.hpe-chevron-left]="!group.open" [class.hpe-chevron-down]="group.open"></i>
+            <h4 class="m-nil">{{ group.heading }}</h4>
```

Comment:
By default they can just specify a `heading` attribute on the accordion-group element and this will add the title in for them, however ngx-bootstrap has no built in support for the chevron. 

I can't modify the existing accordion classes to add a chevron using a pseudo element as the AngularJS version uses the same classes only it has a chevron built in.

I can, if you'd rather provide an additional class they could add to the accordion-group to add a chevron instead that way

Reply:
Sounds good - add a class which includes a chevron on the heading, regardless of how the heading is specified.

Reply:
Will do! Thanks